omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_postgres_validator.rb

```diff
@@ -20,8 +20,10 @@ class PostgresqlPreflightValidator < PreflightValidator
   # we ship. When we bumped the version we ship to 9.6, we haven't yet
   # introduced any changes that _require_ 9.6. So, these constants reflect the
   # actually required PG version.
-  REQUIRED_MAJOR = 9
-  REQUIRED_MINOR = 2
+  REQUIRED_VERSION = 9.6
```

Comment:
If we converted to gem verson these constants could be something like this `Gem::Version.new('13.3')`

Reply:
Done.